scripts/install/overrideEntryPoint.php

```diff
@@ -28,3 +29,12 @@
 $entryPointService->overrideEntryPoint('backoffice', new TaoCeEntrypoint());
 
 $serviceManager->register(EntryPointService::SERVICE_ID, $entryPointService);
+
+$serviceManager = ServiceManager::getServiceManager();
+$entryPointService = $serviceManager->get(EntryPointService::SERVICE_ID);
+
+// add guest login
+$entryPointService->addEntryPoint(
+    new GuestAccess(),
+    EntryPointService::OPTION_PRELOGIN
+);
```

Comment:
It works after a taoUpdate but not on a fresh install... I suspect that it is missing the call to $serviceManager->register(EntryPointService::SERVICE_ID, $entryPointService); after you made the change "addEntryPoint".